corehq/apps/domain/tests/test_deletion_models.py

```diff
@@ -65,6 +66,7 @@
     'util.TransientBounceEmail',
     'registration.AsyncSignupRequest',
     'users.UserHistory',
+    'custom.abdm.models.ABDMUser',
```

Comment:
why should this be ignored?

Reply:
corrected, it should be included in deletion flow.